context/brand-context/default/scss/40-base/normalize.scss

```diff
@@ -39,7 +39,6 @@ main {
 
 h1 {
 	font-size: 2em;
-	margin: 0.67em 0;
```

Comment:
As this is a file not generated by us there is a danger that if we make these changes here that they get overridden by any updates to the normalize code (we updated this fairly recently)

This normalize file also gets used for the `core` level whereas the block spacing does not so we are removing this default spacing from that core experience.

I know it is clunkier but I would be inclined to leave normalize as it is and then reset the margins _before_ including the block spacing file in the `enhanced` view with a message along the lines of "this resets the default margins from the core experience to allow them to be handled by the block spacing for enhanced view"

Reply:
> suggested moving to normalise.scss
We did look at it in the past. Is there are official sass version?

Reply:
Makes sense. Now committed.